html/pfappserver/lib/pfappserver/Form/Config/Syslog/server.pm

```diff
@@ -0,0 +1,71 @@
+package pfappserver::Form::Config::Syslog::server;
+
+=head1 NAME
+
+pfappserver::Form::Config::Syslog::server -
+
+=cut
+
+=head1 DESCRIPTION
+
+pfappserver::Form::Config::Syslog::server
+
+=cut
+
+use strict;
+use warnings;
+use HTML::FormHandler::Moose;
+extends 'pfappserver::Form::Config::Syslog';
+with 'pfappserver::Base::Form::Role::Help';
+
+has_field proto => (
+    type => 'Select',
+    options => [{label => 'udp', value => 'udp'}, {label => 'tcp', value => 'tcp'}],
```

Comment:
Does rsyslog have support for TLS as well ?

Would be a good opportunity to add it